### PR TITLE
Add new step to create definition file

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
$subject

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

**Review by BuddhiWathsala — APPROVED** [review]
LGTM

File: learn/getting-started/writing-a-grpc-service-with-ballerina.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Remove the automatically-created `main.bal` file as you are not going to use it
 
 #### Generating the source code
 
-Execute the following command inside the Ballerina project directory to generate source code related to the service definition.
+Execute the following command inside the same directory where you created the Ballerina project to generate source code related to the service definition.
```
Comment (praneesha) [inline learn/getting-started/writing-a-grpc-service-with-ballerina.md:81]:
```suggestion
Execute the following command inside the same directory in which you created the Ballerina project to generate the source code related to the service definition.
```

File: learn/getting-started/writing-a-grpc-service-with-ballerina.md [hunk 2]
```diff
@@ -162,7 +162,7 @@ Remove the automatically-created `main.bal` file as you are not going to use it
...
```
Comment (praneesha) [inline learn/getting-started/writing-a-grpc-service-with-ballerina.md:165]:
```suggestion
Execute the following command inside the same directory in which you created the Ballerina project to generate the source code related to the service definition.
```

**Comment by praneesha** [comment]
@daneshk - Converted to draft until @anupama-pathirage approves.

**Review by praneesha — APPROVED** [review]

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**